deps/whereami/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+set(WHEREAMI_SOURCES
+	whereami.c
```

Comment:
The `whereami` library is already built as part of RetDec. Cannot we use that build instead of maintaining and building a separate copy?

Reply:
Hm, I guess you are right. I didn't think about it that much when I wanted to propagate the dependency. I guess I got inspired by `jsoncpp` implementation too much. I'll update the `CMakeLists.txt` and get rid of the `whereami` project in `retdec-idapluign`.